src/libraries/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -2068,12 +2062,11 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<System
             void System.IDisposable.Dispose() { }
         }
     }
-    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Might not intended change

Reply:
Looks like the attribute was omitted by the generator